### PR TITLE
Add seed data

### PR DESCRIPTION
**Pull request by larsonkonr**
Updated and created the seed data that we needed

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -2,109 +2,147 @@ class Seed
   def initialize
     build_items
     build_category
-    build_orders
+    #build_orders
```
Comment (bmrsny) [inline db/seeds.rb]: Should we uncomment this and create some order so we can do some testing?

File: db/seeds.rb [hunk 2]
```diff
@@ -5,106 +5,120 @@ def initialize
...
-    item4 = Item.create(title: "Water Tablets", description: "Great way to purify small amounts of water", price: 689, image: open("app/assets/images/water_tablets.jpg"), supplier_id: 2)
-    item5 = Item.create(title: "Water-Beans", description: "Just imagine this was a can of water", price: 899, image: open("app/assets/images/beans.jpg"), supplier_id: 3)
-    item6 = Item.create(title: "Beans", description: "Yummy Beans", price: 500, image: open("app/assets/images/beans.jpg"), supplier_id: 3)
-    item7 = Item.create(title: "Baby Formula", description: "Food for babies", price: 550, image: open("app/assets/images/formula.jpg"), supplier_id: 4)
-    item8 = Item.create(title: "Soup", description: "Canned Soup", price: 750, image: open("app/assets/images/soup.jpg"), supplier_id: 4)
-    item9 = Item.create(title: "Tuna", description: "Chicken of the sea", price: 1200, image: open("app/assets/images/tuna.jpg"), supplier_id: 5)
-    item10 = Item.create(title: "More beans", description: "So many beans!!!", price: 650, image: open("app/assets/images/beans.jpg"), supplier_id: 5)
+    filenames = Dir.glob("app/assets/images/items/*")
```
Comment (bmrsny) [inline db/seeds.rb:26]: That is a cool one. Dir.glob

File: db/seeds.rb [hunk 3]
```diff
@@ -5,106 +5,120 @@ def initialize
...
+  end
 
-	def build_supplier
-		Supplier.create(name: "Joe's big warehouse", slug: "joe-s-big-warehouse", description: "the best", retired: false, address: "1510 blake St. , CO" )
-		Supplier.create(name: "Get Relief Now", slug: "get-relief-now", description: "one stop shop for emergency supplies", retired: false, address: "2020 blake St. , ME " )
-		Supplier.create(name: "Big Supplies", slug: "big-supplies", description: "oversized supplies", retired: false, address: "2000 lake St. , IL " )
-		Supplier.create(name: "To the Rescue", slug: "to-the-rescue", description: "Supplies for those in need.", retired: false, address: "20 bake St. , CA " )
-		Supplier.create(name: "Band-Aid", slug: "band-aid", description: "temporary solutions for big problems", retired: false, address: "2 baker St. , NY " )
-	end
+  def build_supplier
```
Comment (bmrsny) [inline db/seeds.rb:110]: Do we want to use faker gem to create addresses?

File: db/seeds.rb [hunk 4]
```diff
@@ -5,106 +5,120 @@ def initialize
...
-    order10_item7 = OrdersItem.create(item_id: 7, order_id: 10, quantity: 2, subtotal: 1100)
+    statuses = ["ordered", "completed", "paid", "cancelled"]
+    50.times do
+      order_status = statuses.sample
+      Order.create(user_id: rand(1...10), status: order_status, total: rand(1...2000))
+
+      rand(1...10).times do
+        OrdersItem.create(item_id: rand(1...300), order_id: rand(1...50), quantity: 2)
+      end
+    end
```
Comment (scottalexandra) [inline db/seeds.rb:21]: This looks cool!

**Pull request merged**